### ISSUE TITLE
Add a "find-in-document" option

### ISSUE THREAD

**Issue by tmr232** [Moeditor/Moeditor#91]
It would be nice to be able to search for text inside a document.

**Comment by flofriday** [comment]
Yeah a "Find and Replace" would be very helpful, especially at longer files.

**Comment by db2k** [comment]
This should be easy to implement with codemirror.
https://codemirror.net/demo/search.html

**Comment by Menci** [comment]
The new Moeditor will no longer use CodeMirror. I'm switching to cledit.

**Comment by Wamadahama** [comment]
Would the switch make Vim control possible?
